### PR TITLE
Verifying that collatedString works

### PR DESCRIPTION
**Pull request by rjrudin**
Registered it as a known type so that a warning isn't logged either.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,6 +21,7 @@ java {
 
 repositories {
   mavenCentral()
+  mavenLocal()
```
Comment (BillFarber) [inline build.gradle:24]: Did you intend to check in this addition?

Comment (rjrudin) [reply build.gradle:24]: Yes, as I wasn't on the VPN and thus couldn't resolve 7.0-SNAPSHOT from Artifactory. Don't want the VPN to be a requirement for doing stuff.

File: src/test/java/com/marklogic/spark/reader/optic/InferSchemaTest.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ void inferFromAllTdeTypes() throws Exception {
         String columnInfoResponse = readClasspathFile("allTypes-columnInfo-response.txt");
         StructType schema = SchemaInferrer.inferSchema(columnInfoResponse);
 
-        assertEquals(35, schema.size(), "The TDE has 35 columns, and the hidden 'rowid' column that's returned " +
+        assertEquals(36, schema.size(), "The TDE has 35 columns, and the hidden 'rowid' column that's returned " +
```
Comment (BillFarber) [inline src/test/java/com/marklogic/spark/reader/optic/InferSchemaTest.java:39]: Pretty small nit, but does the message also need to be updated?

Comment (rjrudin) [reply src/test/java/com/marklogic/spark/reader/optic/InferSchemaTest.java:39]: Not a nit, the message is wrong. Updating.

**Review by BillFarber — APPROVED** [review]

**Pull request merged**